DQM/DTMonitorModule/python/dtDQMOfflineSources_cff.py

```diff
@@ -14,6 +14,21 @@
                          subSystemFolder = cms.untracked.string('DT')
                          )
 
+# DT digitization and reconstruction
+# Switched to TwinMux
+from EventFilter.L1TXRawToDigi.twinMuxStage2Digis_cfi import *
```

Comment:
hi @jfernan2 -all of the changes in this file need to be moved into the EventFilter package itself. (and the unpackers go into the raw2digi step not the dqm one.